### DISCUSSION TITLE
How to get the number of stiches?

### DISCUSSION THREAD

**Discussion by chrisyelf** [inkstitch/inkstitch#1980] [Q&A] [answered]
Hi, is there a way to see the number of stiches, after I set doc width/height in File > Document properties ?

**Comment by tyrosinase** [comment, marked as answer]
The stitch count will show up almost everywhere Ink/Stitch generates the stitch plan: the Simulator, the PDF, etc. For instance, if you run `Extensions > Ink/Stitch > Simulator / Realistic Preview` then once the simulator starts, the stitch count will be at the end of the progress bar.

**Reply by chrisyelf** [reply]
Thank you, I see it now! Is there a way to get this number from inkscape command line?
